docs/tutorials/clusters/apparmor.md

```diff
@@ -131,7 +131,7 @@ Where `<container_name>` is the name of the container to apply the profile to, a
 specifies the profile to apply. The `profile_ref` can be one of:
 
 * `runtime/default` to apply the runtime's default profile.
-* `localhost/<profile_name>` to apply the profile loaded on the host with the name `<profile_name>`
+* `localhost/<profile_name>` to apply the profile loaded on the host with the name `<profile_name>`.
```

Comment:
Even better would be:
- adding Markdown for an unordered list
- removing the period from this list item and the one above it